dist/backdraft.js

```diff
@@ -1275,8 +1275,18 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
+        if (index===0) {
+          return currentValue
+        }
+        if(currentValue.attr&&currentValue.visible){
```

Comment:
All columns should have attr except for the bulk column. So on this case I check that either they're bulk or have attr and are visible.